entrypoint.sh

```diff
@@ -9,10 +9,14 @@ BUILD_CONTEXT=$5
 # Login to GHCR
 echo ${PERSONAL_ACCESS_TOKEN} | docker login https://ghcr.io -u ${GITHUB_ACTOR} --password-stdin
```

Comment:
Will this work to you if you authenticate with the user `MiniLand1333` but then you push the package to LibreTexts? I guess that since you have permissions in the org it should work but just to be sure :)

Reply:
Yes! This does work:
Successful run: https://github.com/LibreTexts/renderer/actions/runs/757142992
Resulting package: https://github.com/orgs/LibreTexts/packages/container/package/ww-renderer


In contrast, attempting to push to your personal repository from an organization repo/action [results in the following error](https://github.com/LibreTexts/renderer/runs/2346402565?check_suite_focus=true):
`denied: failed_precondition: Improved container support has not been enabled for 'LibreTexts'. Learn more: https://docs.github.com/packages/getting-started-with-github-container-registry/enabling-improved-container-support`. I've already reported this issue to Github support so that they are aware of it and to improve this error message. In case you were wondering, yes I did confirm with them that I did have Improved Container support properly enabled so that wasn't the issue.